### PR TITLE
[BREAKING] Dont use Transfers as Quotes

### PR DESCRIPTION
**Pull request by sentientwaffle**
Use the object found in `schemas/Quote.json` for Quotes instead of `Transfer`.

This will break any code that calls `/quote` directly.

The corresponding sender change is https://github.com/interledger/five-bells-sender/pull/69

### CODE REVIEW

File: schemas/Quote.json [hunk 1]
```diff
@@ -4,6 +4,10 @@
   "description": "A quote object",
   "type": "object",
   "properties": {
+    "source_connector_account": {
+      "description": "the connector's account on the source ledger",
+      "$ref": "Iri.json"
+    },
     "source_ledger": {
```
Comment (emschwartz) [inline schemas/Quote.json]: Is it necessary to return the source and destination ledgers (or the amount that you sent in the request)?

Comment (sentientwaffle) [inline schemas/Quote.json]: No, I suppose not. I'll remove the ledgers. I think leaving both amounts in is simpler, though.

**Comment by emschwartz** [comment]
Note that this PR also removes the same-ledger quoting functionality from the connector

**Comment by sentientwaffle** [comment]
Ya, that logic is moving to the sender.

**Comment by emschwartz** [comment]
LGTM

**Comment by justmoon** [comment]
LGTM

@sentientwaffle is on vacay - so merging this for him.

**Pull request merged**